packages/remix/src/ssr/utils.ts

```diff
@@ -116,3 +116,18 @@ export function getResponseClerkState(requestState: RequestStateWithRedirectUrls
 export const wrapWithClerkState = (data: any) => {
   return { clerkState: { __internal_clerk_state: { ...data } } };
 };
+
+/**
+ * Patches request to avoid duplex issues with unidici
+ * For more information, see:
+ * https://github.com/nodejs/node/issues/46221
+ * https://github.com/whatwg/fetch/pull/1457
+ * @internal
+ */
+export const patchRequest = (request: Request) => {
```

Comment:
@octoper 💡  maybe rename this one to reflect the specific patch? For example `patchRequestDuplex`, or something similar? 🤔

Reply:
Named it `patchRequest` in case we would need to patch other things in the future 😅